src/Orleans.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
```

Comment:
Again, I'll not arg with you anymore since you have the "merge button power" and other people is waiting to have it merged and they are the ones I'm caring about with this work... 

I'll check every single merge from now on and see whether everyone will always keep it stuck on this version... Let's hope you are right... It just doesn't make sense...

Regarding the tests, you as an SDE should know that makes no sense to run or change a test that is not going to bring any benefits since there is no **fork** paths on this code for diff databases and the .sql scripts (which is the only thing that contain database-specific code) doesn't change, **AT ALL**. **All** the guys that created those providers already clearly stated that on Gitter. Anyone who read the code and properly review the changes will clearly note that only run the SQL Server tests will be enough to review/cover this code.

As I said on another comment, I'll run the tests on docker containers for MySQL and Postgre **just to make you happy** and will post here the results, because noone agree with your complains and arguments in that. 

There is nothing personal here. It is just very upset to have no sense complains about something that doesn't change and is being used forever by multiple people. If you want to really properly test every single case, change your CI environment on Jenkins and make all those tests BVT so it can run every single time a PR is created otherwise, we are just wasting precious time that should be focused on other areas that needs attention.